adxl345/example_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package adxl345
```

Comment:
Optional: if you use `package adxl345_test`, the example in the documentation becomes copy-pastable as-is.

Reply:
I'll resolve to commit, this can be done in a follow up and is optional.

Reply:
I've been pushing the change.